src/mantine-hooks/src/use-timeout/use-timeout.ts

```diff
@@ -1,16 +1,16 @@
 import { useRef, useEffect } from 'react';
 
 export function useTimeout(
-  fn: () => void,
+  callback: (callbackParams?: any) => void,
```

Comment:
What if you want to pass more than one argument? Change type to allow any amount of arguments.

Reply:
Pushed changes based on your feedback.